apollo/src/command_config.rs

```diff
@@ -0,0 +1,13 @@
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+/// The [Experimental] Apollo CLI, for supporting all your graphql needs :)
+pub enum Apollo {
```

Comment:
see https://github.com/apollographql/apollo-cli/pull/11/files#diff-78d4a131335698ac4568df51eb2f72fa , I think it would be nice to pull these out as named structs so we can pass the whole struct to command functions, or even so we can just declare `run` as a trait fn on the struct itself.